clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/DeleteGoogleAutoscalingPolicyAtomicOperation.groovy

```diff
@@ -59,6 +60,12 @@ class DeleteGoogleAutoscalingPolicyAtomicOperation extends GoogleAtomicOperation
     this.description = description
   }
 
+  DeleteGoogleAutoscalingPolicyAtomicOperation(DeleteGoogleAutoscalingPolicyDescription description, GoogleClusterProvider googleClusterProvider, GoogleOperationPoller googleOperationPoller) {
```

Comment:
Feels like we're in a bit of an in-between stage here with autowired members and a constructor.  With any luck we can end up with one constructor that takes all the args that are currently autowired, and the description, and the `@Autowired` annotation disappears from the members.

Reply:
It may not be possible to have only 1 constructor, because it seems [here](https://github.com/spinnaker/clouddriver/blob/ff7c74e1c522fdc3239f06c956dbd0bc5281d214/clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/converters/DeleteGoogleAutoscalingPolicyAtomicOperationConverter.groovy#L33) other required details are not available.

Reply:
`DeleteGoogleAutoscalingPolicyAtomicOperationConverter` is a `@Component`, so can't it autowire the additional members it needs?

Reply:
Updated the code.